devops/e2e/cloudtest/build-infra.json

```diff
@@ -0,0 +1,20 @@
+// Base Image: 1ES PT - Ubuntu 22.04 - vNext
+{
+    "artifacts": [
+        {
+            "name": "linux-azcli"
+        },
+        {
+            "name": "linux-dotnet-sdk",
+            "parameters": {
+                "version": "6.0"
+            }
+        },
+        {
+            "name": "linux-bash-command",
+            "parameters": {
+                "command": "apt list --installed"
```

Comment:
What is the objective of this?

Reply:
So we know what packages are installed on image when published. An SBOM of sorts so we can easily see it in the imaging logs.